src/App.tsx

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    function isIncludes(str: string, substr: string) {
+      return str.includes(substr);
+    }
+
+    const isCoincidence = isIncludes(
```

Comment:
move to normalize these strings to the function `isIncludes`
Also, move this func from the component